ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -49,7 +49,7 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         func backgroundColor(fluentTheme: FluentTheme) -> UIColor {
             switch self {
             case .header, .footer, .headerPrimary:
-                return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background1])
+                return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background2])
```

Comment:
Shouldn't this be accessing `TableViewTokens`?

Reply:
I believe tokenizing TableViewHeaderFooterView is a different work item, and hence outside of the scope of this change.